lib/content-services/src/lib/common/services/rendition.service.ts

```diff
@@ -90,7 +90,7 @@ export class RenditionService {
     private async waitRendition(nodeId: string, renditionId: string, retries: number): Promise<RenditionEntry> {
         const rendition = await this.renditionsApi.getRendition(nodeId, renditionId);
 
-        if (this.maxRetries < retries) {
+        if (this.maxRetries >= retries) {
```

Comment:
can you write unit test which will prevent that issue in the future?

Reply:
@AleksanderSklorz sure, I have added a test in new commit